contexts/network-settings.tsx

```diff
@@ -219,7 +220,7 @@ export const NetworkSettingsProvider = ({ children }) => {
           return undefined;
 
         // Reserved names
-        if (/bepro|taikai/gi.test(value))
+        if (/bepro|taikai/gi.test(value) || STATIC_URL_PATHS.includes(value?.toLowerCase()))
```

Comment:
it can't be "includes" because I might want to make the "moshmage network" and this won't allow it, but the path is just /.../network/.../ we might need the "is this a route" thing after all

Reply:
include is only true if the route has the same name. in this example it allows "moshmage network" and any other that has a name different from the defined constants